src/2dplane/ObstacleGrid.cpp

```diff
@@ -24,6 +25,30 @@ Vector2i ObstacleGrid::gridSquareForLocation(const Vector2f &loc) const {
                     loc.y() / height() * discretizedHeight());
 }
 
+float ObstacleGrid::nearestObstacleDist(const Vector2f &state) const {
```

Comment:
It may be better to make maxDist be a parameter to this method rather than a property on the ObstacleGrid class because maxDist isn't really an intrinsic part of a grid of obstacles, it's just a limit to how far out we want to search in certain cases.  Thoughts?